File: Solutions/Easy/1018. Binary Prefix Divisible By 5/Solution.cs
public class Solution {
    public IList<bool> PrefixesDivBy5(int[] nums) {
        List<bool> result = new List<bool>();
        int bTen = 0;
        foreach(int num in nums){
            bTen <<= 1;
            bTen += num;
            bTen %= 5;
            result.Add((bTen == 0));
        }
        return result;
    }
}

File: Solutions/Easy/1018. Binary Prefix Divisible By 5/Solution.py
class Solution:
    def prefixesDivBy5(self, nums: List[int]) -> List[bool]:
        result: List[int] = []
        rem: int = 0
        for num in nums:
            rem <<= 1
            rem += num
            rem %= 5
            result.append((rem == 0))
        return result